### ISSUE TITLE
delete QoStestClient code?

### ISSUE THREAD

**Issue by xu2004jie** [open-webrtc-toolkit/QoSTestFramework#19]
Hello Team,
     Delete QoStestClient code ? how build this project ? and how test webrtc native client?  Thank you.

Best Regards
Jack Xu

**Comment by AliceCodeZhang** [comment]
Yes, Please following https://github.com/open-webrtc-toolkit/owt-client-native to build the code
